provisioning/playbook.yml

```diff
@@ -0,0 +1,10 @@
+---
+- hosts: localhost
+  user: root
+
+  vars_files:
+    - vars.yml
+
+  roles:
+    - facts
+    - nodejs
```

Comment:
No newline at the end of the file, same as "provisioning/requirements.yml" and "provisioning/vars.yml".